docs/learn/overview/intro.mdx

```diff
@@ -2,4 +2,28 @@
 slug: /
 ---
 
-# Welcome to the Lit Protocol Docs
\ No newline at end of file
+# Welcome to the Lit Protocol Docs
+
+You are viewing the _Learn_ section of the Lit Protocol documentation. This section is aims to teach the various Lit Protocol concepts and how to use them.
+
+If you are looking for information on how to build on Lit Protocol, please visit the [Build section](/build) of the documentation.
+
+## Getting Started with Lit
+
+If you are completely new to Lit, we recommend starting with the [What is Lit Protocol](./what-is-lit) page, followed by the [How Lit Protocol Works](./how-it-works) page. These pages will provide you with a high-level overview of Lit Protocol and what you can do with it.
+
+## Core Concepts
+
+Once you have a high-level overview of Lit Protocol, you can dive into the core concepts. These concepts are the building blocks of Lit Protocol and are prerequisites when building with Lit.
+
+- [The Lit SDK](./lit-sdk/overview) - a collection of libraries that make it easy to interact with Lit Protocol.
+- [Authentication](../authentication/overview) - proving your identity to the Lit Network.
+- [Paying for Usage](../pay-for-usage/overview) - how to pay for the resources you use on the Lit Network.
+
+## How You Can Use Lit
+
+After you have a good understanding of the core concepts, you can explore how you can make use of Lit Protocol in your project.
+
+- [Signing Data](../signing-data/overview) - leveraging programmable private keys to sign data and/or transactions without exposing the private key.
```

Comment:
Would we want to say programmable private keys or programmable key pairs here? I guess there isn't much difference